kalite/distributed/management/commands/kaserve.py

```diff
@@ -149,7 +149,11 @@ def handle(self, *args, **options):
             call_command("runserver", "%s:%s" % (options["host"], options["port"]))
         else:
             del options["production"]
+            addresses = get_ip_addresses(include_loopback=False)
             sys.stdout.write("To access KA Lite from another connected computer, try the following address(es):\n")
-            for addr in get_ip_addresses():
+            for addr in addresses:
                 sys.stdout.write("\thttp://%s:%s/\n" % (addr, settings.USER_FACING_PORT()))
+            sys.stdout.write("To access KA Lite from this machine, try the following address:\n")
+            sys.stdout.write("\thttp://127.0.0.1:%s/\n" % settings.USER_FACING_PORT())
```

Comment:
There should kind of be a TODO on this, I don't believe that 127.0.0.1 is always the loopback... there could be one or more, and they could be different.